Common/Data/Custom/PsychSignal/PsychSignalSentiment.cs

```diff
@@ -122,7 +142,7 @@ public override BaseData Reader(SubscriptionDataConfig config, string line, Date
 
                 return new PsychSignalSentiment
                 {
-                    Time = timestamp,
+                    EndTime = timestamp,
```

Comment:
A note for posterity... in LEAN, the `EndTime` is the *only* time that matters. We don't (or at least _shouldn't_) use the `Time` property for anything in the engine.